### PR TITLE
Paypal integration docs

### PR DESCRIPTION
**Pull request by anirudh-tantry**

### CODE REVIEW

File: docs/.vuepress/public/assets/images/paypal-integration/payment_configuration.png
Comment (piyushsinghania) [inline docs/.vuepress/public/assets/images/paypal-integration/payment_configuration.png:1]: For images you might want to use https://pika.style/

File: docs/readme.md [hunk 1]
```diff
@@ -1 +1,19 @@
-# Hello Formester
+---
+home: true
+heroText: Formester Docs
+tagline: To start using Formester, check out our Guide
+actionText: See our guide →
+actionLink: ./guide.md
```
Comment (piyushsinghania) [inline docs/readme.md:6]: Somehow I am not able to find the actionText and actionLink on the home page

image


https://deploy-preview-1--formester-docs.netlify.app/

File: docs/paypal-integration.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+Navigate to the Formester app and access the [Payments section](https://app.formester.com/payments). From the available payment options, choose PayPal as the preferred payment gateway.
+
+
+### Step 2: Fill in the required details
+
+![Formester Payments Page](/assets/images/paypal-integration/formester_paypal_integration.png)
+
+Provide your PayPal email address along with the Client ID and Secret Key you obtained earlier. This information will enable Formester to securely communicate with PayPal and process payments seamlessly.
+
+### Step 4: Integrate
```
Comment (piyushsinghania) [inline docs/paypal-integration.md]: `Step 3` is missed by chance. Probably it should be `Step 3`

**Review by piyushsinghania — APPROVED** [review]

File: docs/.vuepress/public/assets/images/paypal-integration/payment_configuration.png
Comment (anirudh-tantry) [reply docs/.vuepress/public/assets/images/paypal-integration/payment_configuration.png:1]: All the screenshots are not full screen. Can use this @piyushsinghania?

File: docs/readme.md (hunk 1 above)
Comment (anirudh-tantry) [reply docs/readme.md:6]: Yes action link is not working. I tried it many times its not showing the button

**Review by piyushsinghania — APPROVED** [review]

**Pull request merged**